pkgs/development/interpreters/nickel/default.nix

```
@@ -24,6 +24,9 @@ rustPlatform.buildRustPackage rec {
     python3
   ];
 
+  # Disable checks on Darwin because of issue described in https://github.com/tweag/nickel/pull/1454
+  doCheck = !rustPlatform.rust.rustc.stdenv.isDarwin;
```

```suggestion
  doCheck = !stdenv.isDarwin;
```
and add `stdenv` to the arguments list.